### DISCUSSION TITLE
Australian bank accounts - Spectre import transaction matching

### DISCUSSION THREAD

**Discussion by mpfl** [firefly-iii/firefly-iii#5400] [Firefly III] [answered]
Australian bank accounts don't tend to have IBANs. We work on our own arcane system: each bank account has a six digit "BSB" (Bank, State, Branch) to go with the account number (which is only guaranteed to be unique within a bank). So a ful bank account number might look like 123-456 2895873.

I have more bank accounts than sense and would like Firefly to match all the transfers I make, rather than noting them as unrelated withdrawals and deposits.

Does anyone know what information I need to put where so that Firefly III will match the transfer transactions as I import them from Spectre? I've tried putting the BSB and account number combination in the IBAN field but the error logic on that field prevents me from doing it.

**Comment by JC5** [comment]
You could use the account number field perhaps?

**Comment by mpfl** [comment, marked as answer]
Ah, I just found the Spectre dashboard where I can inspect the metadata for all the transactions that have been imported. Unfortunately transfers don't have any metadata about the destination account.

**Reply by JC5** [reply]
If they have a description you can use, you can set up rules to manage what happens to each transaction. You can change the transaction type, the destination account, etc.

**Reply by mpfl** [reply]
They do not, unfortunately. The full text of the description is made by the person making the transfer. I could probably put some kind of identifier in the descriptions myself as I make the transfers, but I am not the only person who uses these accounts.

**Reply by JC5** [reply]
I'm sorry, in those cases it's difficult for the importer (or Firefly III) to do a lot about it :(

**Reply by mpfl** [reply]
Yeah, I should just get used to the fact that the banking industry in Australia is archaic and we can't have nice things. Oh well.

Thanks for your support!

**Reply by JC5** [reply]
At least you have spiders!

![Spiders](https://media.giphy.com/media/119mPbgfgGnXig/giphy.gif)
